glacier/src/actions/add-field.ts

```diff
@@ -0,0 +1,8 @@
+import {ReduxStandardAction} from "./";
+import {Field} from "../model";
+
+export type AddFieldAction = ReduxStandardAction<"ADD_FIELD", Field[]>;
+
+export function createAddFieldAction(fields: Field[]): AddFieldAction {
+    return {type: "ADD_FIELD", payload: fields};
```

Comment:
I'm not sure that's necessary. I can include it, I guess. I thought table and field name would be sufficient.